### ISSUE TITLE
Error in ManageDirectory.

### ISSUE THREAD

**Issue by damsfx** [OFFLINE-GmbH/oc-bootstrapper#36]
Oc-bootstrapper 0.5.2 !!  (damned, it was a so shine tools)   

Getting error while creating project with `october init`  on Windows OS :
````
$ october init test
Creating project directory...
Updating template files...
Creating default october.yaml...
PHP Warning:  copy(P:\_Sites\tests\C:\Users\me\AppData\Roaming\Composer\october\october.yaml): failed to open stream: Invalid argument in C:\Users\me\AppData\Roaming\Composer\vendor\offline\oc-bootstrapper\src\Util\ManageDirectory.php on line 24
````
It's look like the two arguments are not separated !!

**Comment by damsfx** [comment]
One more thing .... 
Downgrading offline/oc-bootstrapper (v0.5.2 => v0.5.0) solved the issue.

**Comment by tobias-kuendig** [comment]
This issue should be fixed via 93cddac by @mtellenbach and is released as 0.5.3.

Please report back if you experience any further issues. Thanks for the bug report!
